assets/scss/app.scss

```diff
@@ -0,0 +1,127 @@
+$path: "../img/";
```

Comment:
we are defining a variable $path but not using it - below we are using a relative path to `../images`